jcl/src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -371,6 +372,9 @@ MethodType accessModeType(Class<?> receiver, Class<?> type, Class<?>... args) {
 		this.coordinateTypes = coordinateTypes;
 		this.handleTable = handleTable;
 		this.modifiers = modifiers;
+/*[IF Java14]*/
+		this.exact = false;
+/*[ENDIF] Java14 */
```

Comment:
`Java 14` is out of service, I would suggest use `Java15` instead. The block containing `final boolean exact;` can be decorated with `/*[IF Java15]*/` as well and combined with the immediate following `/*[IF Java15]*/` block.

Reply:
Refer to https://github.com/eclipse/openj9/pull/11182#discussion_r523298021.

Reply:
Now, the all changes are only exposed in `Java16`.